lib/shoulda/matchers/active_record/association_matchers/dependent_matcher.rb

```diff
@@ -34,6 +33,18 @@ def matches?(subject)
           def option_verifier
             @option_verifier ||= OptionVerifier.new(subject)
           end
+
+          def correct_for?(dependent=dependent)
```

Comment:
The original reason was that there is a `dependent` method and therefore the argument wouldn't be needed, but I didn't like how the code read if the argument wasn't passed. But, I just made the change that you suggested.